include/boxfort.h

```diff
@@ -277,29 +281,33 @@ struct bxf_start_params {
 
 #define BXF_FOREVER INFINITY
 
-typedef const struct bxf_start_params *bxf_start_params;
+typedef const struct bxf_start_params_s *bxf_start_params;
 
 #define bxf_start(Instance, Sandbox, ...) \
     (bxf_start_struct((Instance),         \
     (Sandbox),                            \
-    &(struct bxf_start_params) { .bxfi_sentinel_ = 0, __VA_ARGS__ }))
+    &(struct bxf_start_params_s) { .bxfi_sentinel_ = 0, __VA_ARGS__ }))
 BXF_API int bxf_start_struct(bxf_instance **instance,
-        bxf_sandbox *sandbox, bxf_start_params params);
+        bxf_sandbox *sandbox, struct bxf_start_params_s *params);
```

Comment:
nit: this doesn't need to change

Reply:
to clarify: `bxf_start_params` and `bxf_spawn_params` are const pointers, while the change makes them mutable. This changes slightly the API unnecessarily.